Quest: undefined reference to 'blis_abort'

hello，
        After installing the blis with configure 'zen', I tryied #include "blis.h" in my code. But, when I compile it with command "gcc -fopenmp -lm -lblis *.c -o a.out", I receive the error of ld as written by my title.
        Besides, when I tried commands :  "gcc -fopenmp -c *.c -o *.o" & "gcc *.o $(PREFIX)/lib/zen/libblis.a -fpenmp -lm -o a.out", I make it successfully. How to solve  or explain this question?

@devinamatthews pity to say that yes, it still happens, In function `bli_gemmsup_ref_var1n2m_opt_cases': test.c:(.text+0x11661): undefined reference to `bli_abort'

./test.c:3:12: error: conflicting types for ‘bli_gemmsup_ref_var1n2m_opt_cases’
 extern int bli_gemmsup_ref_var1n2m_opt_cases();
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from ./test.c:1:0:
/home/test/blis/include/blis/blis.h:35841:18: note: previous definition of ‘bli_gemmsup_ref_var1n2m_opt_cases’ was here
 BLIS_INLINE void bli_gemmsup_ref_var1n2m_opt_cases
                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Oh, the one you sent to me tell me that : gcc -fopenmp ./test.c -o a.out -lblis -lm
/tmp/ccRmriX4.o: In function `main':
test.c:(.text+0x15): undefined reference to `bli_gemmsup_ref_var1n2m_opt_cases'
collect2: error: ld returned 1 exit status

--- a/frame/base/bli_error.h
+++ b/frame/base/bli_error.h
@@ -39,7 +39,7 @@ BLIS_EXPORT_BLIS void     bli_error_checking_level_set( errlev_t new_level );
 BLIS_EXPORT_BLIS bool_t    bli_error_checking_is_enabled( void );
 
 void      bli_print_msg( char* str, char* file, guint_t line );
-void      bli_abort( void );
+BLIS_EXPORT_BLIS void      bli_abort( void );
 
 char*     bli_error_string_for_code( gint_t code );
 

